metrics/cli/test/sentry/model/sentry_release_test.dart

```diff
@@ -0,0 +1,51 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:cli/sentry/model/sentry_project.dart';
+import 'package:cli/sentry/model/sentry_release.dart';
+import 'package:test/test.dart';
+
+// ignore_for_file: avoid_redundant_argument_values
+
+void main() {
+  group("SentryRelease", () {
```

Comment:
Should we test that 2 instances with the same parameters are equal? The same suggestion for all classes extending `Equatable`.